projects/core/src/cms/page/config/default-page-meta.config.ts

```diff
@@ -29,5 +34,13 @@ export const defaultPageMetaConfig: PageMetaConfig = {
         method: 'resolveRobots',
       },
     ],
+    options: {
+      canonicalUrl: {
+        forceHttps: true,
+        forceWww: true,
+        removeQueryParams: true,
+        forceTrailingSlash: true,
```

Comment:
[medium] IMHO those are opinionated and we shouldn't enable them by default: `forceWWW`, `forceTrailingSlash`. I feel the same, but more relaxed, for `removeQueryParams`. What do you think?

Reply:
I disagree, I've seen things that are not enabled by default are easily missed by projects. That being said, I like the idea to disable specific resolvers all together (similar to `disabledInCsr`). We could then disable the canonical URL completely in 3.x releases.